hi-ml-histopathology/src/histopathology/configs/classification/DeepSMILEPanda.py

```diff
@@ -139,11 +144,24 @@ def setup(self) -> None:
             self.downloader = self.download_ssl_checkpoint(innereye_ssl_checkpoint_binary)
         BaseMILSlides.setup(self)
 
+    def get_dataloader_kwargs(self) -> dict:
+        return dict(
+            multiprocessing_context="spawn",
```

Comment:
Does "spawn" now still make a difference?

Reply:
I haven't checked tbh. but maybe it's safer to use spawn in case we want to train with a higher resolution at some point?

Reply:
Yes, we can leave spawn if we are sure there is no negative impact on the training with level 1 - that's the most important thing at this point.